package.json

```diff
@@ -1,19 +1,23 @@
 {
   "name": "obsidian-rollover-daily-todos",
-  "version": "0.9.7",
-  "description": "This plugin rolls over incomplete TODOs to the next daily note for Obsidian (https://obsidian.md)",
+  "version": "1.1.0",
+  "description": "This Obsidian.md plugin rolls over incomplete TODOs from the previous daily note to today's daily note. (https://obsidian.md)",
   "main": "main.js",
   "scripts": {
     "dev": "rollup --config rollup.config.js -w",
-    "build": "rollup --config rollup.config.js"
+    "build": "rollup --config rollup.config.js --environment BUILD:production",
+    "deployLocally": "rollup --config rollup.config.js --environment BUILD:production && cp main.js ../../../../My_Obsidian/.obsidian/plugins/obsidian-rollover-daily-todos/main.js"
```

Comment:
This looks specific to your obsidian location relative to the location of this plugin. Could you instead pass the path to obsidian as an argument here?

Reply:
Forgot to remove this. But I just removed it, and added a new js file to the `.gitignore` which does the same thing, and of course added that to my local repo.